ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -718,6 +721,41 @@ def trunc(x):
     return ivy.trunc(x)
 
 
+@with_supported_dtypes(
+    {"0.4.14 and below": ("float32", "float64", "int32", "int64")}, "jax"
+)
+@to_ivy_arrays_and_back
+def unwrap(p, discont=None, axis=-1, period=2 * ivy.pi):
+    p = ivy.asarray(p)
+    dtype_str = str(p.dtype)
+    if "int" in dtype_str:
```

Comment:
Instead of multiple lines of code and if-elif statements, we could use a dict mapping similar to jax [here](https://github.com/google/jax/blob/c3e73c67aa310aff2da46f251bf52352ee094a7f/jax/_src/dtypes.py#L162).

Reply:
Done, replaced with dict mapping